### ISSUE TITLE
Still error on playerdata

### ISSUE THREAD

**Issue by broken1arrow** [Zrips/CMI#2738]
**Description of issue:**
playerdata gets Timer reset, now is start around 19/20 hours (after ca 1 hour He play).  
Generate new playerdata for this player same result (was zero and now 19 hour and cunt up). 

We have test server and it work fine this player not get data reset (it use 8.6.17.1). Use same world file (copy from main server)

If you need files, i´m send private (want not send it public)
---

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
#######################################
## REPLACE WITH RELEVANT CONFIG SECTION  ##
#######################################
```

---

**Cmi Version (using`/cmi version`):** 
8.6.18.7
**Server Type (Spigot/Paperspigot/etc):** 
paper
**Server Version (using `/ver`):** 
87 (15.2)
**Relevant plugins (Delete if this isn't needed):**
CMI

**Comment by Zrips** [comment]
i need more information on this one. What you mean by timer reset? Playtime? If so, then check player statistics files, as those one track player playtime too. And check from where CMI gets its statistics.

**Comment by broken1arrow** [comment]
Yes CMI read data, but in playerdata file the tick change and CMI read the new value.
If i change it so change it on cmi too, but only around one hour and change back too between 19-1day playtime agen. In this chase is change from zero playtime too 19 hours.

So something happen with playerdata.

**Comment by broken1arrow** [comment]
Is where cmi take time data (blue line to mark). Is in world (we call it midgardsaga)/ playerdata, not in advancements (stats).
![time](https://user-images.githubusercontent.com/16368286/73666981-e7c0b080-46a3-11ea-822b-f58a40ea58e3.PNG)

In advancements stats is it 61 days. So if we cant fix the time is hole ranking system useless.
![time2](https://user-images.githubusercontent.com/16368286/73667179-2b1b1f00-46a4-11ea-9179-8f41b7972635.PNG)

**Comment by broken1arrow** [comment]
Even reward system get broken

![Reward](https://user-images.githubusercontent.com/16368286/73667803-1f7c2800-46a5-11ea-981d-c4c24f9126be.PNG)

When i use the command it start working agen, but he has lose all reward some he don't claim (it come back if I change the time). But like I sad is always get reset ether zero ore 20 hours ca.
![reward2](https://user-images.githubusercontent.com/16368286/73667903-4c303f80-46a5-11ea-8ad0-57648ac95c49.PNG)

**Comment by broken1arrow** [comment]
I´m try the world whit no CMI and CMI have standard setting (but use my computer to host server for test). Same player log in no issue.

So i´m guess error in database or settings. So wear lock for problems first?

**Comment by broken1arrow** [comment]
@Zrips 

So can you help or not?
I´m have come closer to solve this issue. 


So i´m fins out if player go AFK it reset playtime to 18 hour 57 minutes and it is settings (config), for one player get issue. 

I´m don´t now way is only one player get this bug. If you read this you need take serious look on your code, for plugin shall not mess with data like this.

Is only second between this images.
![2020-02-08_09 17 23](https://user-images.githubusercontent.com/16368286/74081889-1350eb80-4a54-11ea-83b3-497abcf1a758.png)
![2020-02-08_09 17 27](https://user-images.githubusercontent.com/16368286/74081890-13e98200-4a54-11ea-882b-110fe6b42a6c.png)

**Comment by broken1arrow** [comment]
You probably have much to do, but no help even when you get setting file.

So i´m do bug searching whit not knowing how you make the plugin and how you write here is sound at you don´t make CMI alone.

I´m find the error in the file (one setting you not shall set on true), can mention he is first player in database (first place in the database).

I can tell you the error, if you are interested to fix your plugin.

**Comment by Zrips** [comment]
Well if you have error in your logs related to CMI then this bug report should have started with it. Next, there are 2 places from where player playtime can be taken: player data file in main world folder or player statistic file in statistics folder. Both of them can be same value or completely different if you have modified one of them in the past. CMI can pick from one of them and you can define which one you want in config file under PlayTimeFromStats config section

**Comment by broken1arrow** [comment]
@Zrips 

No error in console (if it was error in console i´m ofc send it to you), I and a friend come up what it was is trigger it and i try it out the bug (trial and error). 

Is one setting in settings file some trigger it. This setting StopPlayTime: can we don't have on true like the file a send you. One player get infected, and it change in world/playerdata and time set to 18 hours and 57 minuter, sometime 19 hours.

No mater what we set other AFK setting and it not any other plugin (i´m use clean install).

**Comment by LogGits** [comment]
@broken1arrow are you saying that you modified a specific player's playtime and following that edit, their stats didn't update for rankups and the reward system?

**Comment by broken1arrow** [comment]
@LogGits 

When I manually change the time (After I fund it was  StopPlayTime in config), so has it work after. Before every time i change the time it get reset to 19 hours (if he go AFK) and how i´m start even to trace down the problem (CMI change playerdata for this player, even before i´m touch this file).

The problem start in middle of December (it was not easy to solve this and Zrips give not much clue).

Hi is first in database on cmi to (can be one of the problem), second he is admin (some permissions is buggy perhaps).
